library/src/main/java/com/datatorrent/lib/appdata/schemas/SchemaUtils.java

```diff
@@ -490,5 +493,54 @@ public static JSONObject createJSONObject(Map<String, String> map)
     return jo;
   }
 
+  /**
+   * This is a helper method which converts the given {@link JSONArray} to a {@link List} of Strings.
+   *
+   * @param jsonStringArray The {@link JSONArray} to convert.
+   * @return The converted {@link List} of Strings.
+   */
+  public static List<String> getStringsFromJSONArray(JSONArray jsonStringArray) throws JSONException
+  {
+    List<String> stringArray = Lists.newArrayList();
```

Comment:
Can Lists.newArrayListWithCapacity() not be used since the the length of List is known?